File: c/PRAK501-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d) {
    int max1 = (a > b) ? a : b;
    int max2 = (c > d) ? c : d;
    return (max1 > max2) ? max1 : max2;
}    

int main() {
        int a, b, c, d;
        scanf("%d %d %d %d", &a, &b, &c, &d);
        int hasil = MaxBilangan(a, b, c, d);
        printf("%d", hasil);
        return 0;
}

File: python/PRAK501-2310817310004-MSAMILRENDYNORSALEH.py
def MaxBilangan(a, b, c, d):
    max1 = a if a > b else b
    max2 = c if c > d else d
    return max1 if max1 > max2 else max2

if __name__ == "__main__":
    a,b,c,d = map(int, input().split())
    hasil = MaxBilangan(a,b,c,d)
    print(hasil)
